web/src/components/Input.js

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Box, Input, Label } from 'rebass';
+
+const InputHolder = ({ input, label, type, meta: { touched, error } }) => (
+  <Box mb={3}>
+    <Label>{label}</Label>
+    <Input {...input} type={type} />
```

Comment:
Maybe getting a little ahead of us here, but I'm wondering if we should generate a unique ID here for the `Input` and then assign the `Label`'s `htmlFor` prop to that value?  It'd be pretty nice to not have to worry about the IDs.

Just threw together an example of how it might work to be sure the `id` and `htmlFor` props would pass through Rebass, and they do.  😄

Reply:
yes, those definitely should be in here -- will add, thx